app/pages/sections/order-items/catalogue-solutions/select/routes.js

```diff
@@ -294,5 +311,75 @@ export const catalogueSolutionsSelectRoutes = (authProvider, addContext, session
     return res.render('pages/sections/order-items/catalogue-solutions/select/date/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.get('/solution/price/:priceType/:provisioningType', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
+    const { orderId, priceType, provisioningType } = req.params;
+    const itemName = sessionManager.getFromSession({
+      req, key: sessionKeys.selectedItemName,
+    });
+    const selectedPrice = sessionManager.getFromSession({
+      req, key: sessionKeys.selectedPrice,
+    });
+    const quantity = sessionManager.getFromSession({
+      req, key: sessionKeys.selectedQuantity,
+    });
+    const estimationPeriod = sessionManager.getFromSession({
+      req, key: sessionKeys.selectEstimationPeriod,
+    });
+    const formData = {
+      quantity,
+      selectEstimationPeriod: estimationPeriod,
+    };
+    const context = await getProvisionTypeOrderContext({
+      orderId,
+      orderItemType: 'solution',
+      selectedPrice,
+      itemName,
+      formData,
+    });
+    logger.info(`navigating to order ${orderId} catalogue-solutions ${provisioningType} form`);
+    if (priceType === 'flat' && provisioningType === 'patient') {
+      return res.redirect(`${config.baseUrl}/organisation/${orderId}/catalogue-solutions/neworderitem`);
+    }
+    return res.render(`pages/sections/order-items/catalogue-solutions/order-item/${priceType}/template.njk`, addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
+  }));
+  router.post('/solution/price/:priceType/:provisioningType', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
```

Comment:
Blank line between the route handler functions.

Reply:
Added blank line.